Client/Frontend/Popup/PopupView.swift

```diff
@@ -87,6 +87,9 @@ class PopupView: UIView, UIGestureRecognizerDelegate {
     
     var dialogWidth: CGFloat {
         get {
+            if UIDevice.current.userInterfaceIdiom == .pad {
+                return min((applicationWindow?.bounds.width ?? UIScreen.main.bounds.width) - padding * 2.0, kPopupDialogMaxWidth)
```

Comment:
Is it possible to replace the layout solution to not use these widths at all (and instead use its parents width)